docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -343,10 +343,13 @@ Release
 
    .. note:: The following instructions use `virtualenvwrapper`_ but you can use `virtualenv`_ directly if you prefer.
 
+   .. note:: The final command ensures that setuptools is a version that does not normalize version numbers according to PEP440.
```

Comment:
I think I'll leave that link out. Knowing that we might one day move to PEP440 versioning is not necessary to understand the note, which is itself merely a distraction to getting on with releasing.